Gemfile

```diff
@@ -3,34 +3,38 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.4.1'
 
-gem 'rails', '~> 5.2.0.rc1'
-gem 'pg', '>= 0.18', '< 2.0'
+gem 'bootsnap', '>= 1.1.0', require: false
+gem 'devise'
 gem 'news-api'
-gem 'stripe-rails'
+gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
+gem 'pundit'
+gem 'rails', '~> 5.2.0.rc1'
 gem 'sass-rails', '~> 5.0'
-gem 'uglifier', '>= 1.3.0'
+gem 'stripe-rails'
 gem 'turbolinks', '~> 5'
-gem 'bootsnap', '>= 1.1.0', require: false
-gem 'devise'
+gem 'uglifier', '>= 1.3.0'
 
 group :development, :test do
   gem 'chromedriver-helper'
-  gem 'selenium-webdriver'
+  gem 'coveralls', require: false
+  gem 'cucumber-rails', require: false
+  gem 'database_cleaner'
   gem 'factory_bot_rails'
+  gem 'launchy'
   gem 'pry-byebug'
   gem 'pry-rails'
+  gem 'pundit-matchers', '~> 1.4.1'
   gem 'rspec-rails'
+  gem 'selenium-webdriver'
   gem 'shoulda-matchers'
-  gem 'cucumber-rails', require: false
-  gem 'database_cleaner'
-  gem 'launchy'
-  gem 'coveralls', require: false
+  gem 'chromedriver-helper'
+  gem 'selenium-webdriver'
 end
 
 group :development do
-  gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'web-console', '>= 3.3.0'
```

Comment:
Why do you add this dependency?

Reply:
We didn't add it, we just sorted the gem's in alphabetical order 👍

Reply:
Okay